src/lib/html.ts

```diff
@@ -36,3 +36,20 @@ export const removeClass = (el: HTMLElement, className: string): void => {
 export function $<T extends HTMLElement = HTMLElement>(query: string): T {
   return document.querySelector(query) as T
 }
+
+/**
+ * Returns true if an element is found matching a given css selector.
+ * @param query
+ */
```

Comment:
add `tagName` @param

Reply:
gonna remove the tagName param, cause we can add the tag name in the CSS selector.